services/wallet/network/network.go

```diff
@@ -0,0 +1,185 @@
+package network
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+type Network struct {
+	ChainID                uint64 `json:"chain_id"`
+	ChainName              string `json:"chain_name"`
+	RPCURL                 string `json:"rpc_url"`
+	BlockExplorerURL       string `json:"block_explorer_url,omitempty"`
+	IconURL                string `json:"icon_url,omitempty"`
+	NativeCurrencyName     string `json:"native_currency_name,omitempty"`
+	NativeCurrencySymbol   string `json:"native_currency_symbol,omitempty"`
+	NativeCurrencyDecimals uint64 `json:"native_currency_decimals"`
+	IsTest                 bool   `json:"is_test"`
+	Layer                  uint64 `json:"layer"`
+	Enabled                bool   `json:"enabled"`
+}
```

Comment:
should we use camelCase instead of snake_case in these annotations?

Reply:
Will do, TBH I don't know what I should use I've seen both. :confused: